README.md

```diff
@@ -1052,15 +1052,18 @@ Manages configuration of a BGP Address-family instance.
 |----------|:------------------:|:----------------------:|
 | N9k      | 7.0(3)I2(1)        | 1.1.0                  |
 | N3k      | 7.0(3)I2(1)        | 1.1.0                  |
+| N5k      | 7.3(0)N1(1)        | 1.3.2                  |
 | N6k      | 7.3(0)N1(1)        | 1.2.0                  |
 | N7k      | 7.3(0)D1(1)        | 1.2.0                  |
 
 #### <a name="cisco_bgp_af-caveats">Caveats</a>
 
 | Property | Caveat Description |
 |:--------|:-------------|
-| `additional_paths_install` | Not supported on N3k, N9k |
-| `advertise_l2vpn_evpn`     | Not supported on N3k, N6k |

```

Comment:
Again, I would clarify it with AF, and I would pair them:

| Property | Caveat Description |
|:------------|:--------------------------|
| `additional_paths_install` | Not supported on N3k, N9k|
| `advertise_l2vpn_evpn`     | Not supported on N3k, N6k|
| address-family `l2vpn/evpn`| Module Minimum Version `1.3.2`<br>OS Minimum Version `7.0(3)I3(1)`<br>Not supported on N3k |